### ISSUE TITLE
Authentication - Redirect users to Sign in screen

### ISSUE THREAD

**Issue by vusters** [EnsembleUI/ensemble#1102]

**Comment by kmahmood74** [comment]
yes - I think the way to do is to specify at the app level what to do when API returns a specific header or content. The HTTP header for unauthorized access is 401 but some apps send errors in the body with a 200 error. Let the server return it instead of proactively checking.
